draft-ietf-quic-multipath.md

```diff
@@ -1315,6 +1315,8 @@ processing of a RETIRE_CONNECTION_ID frame is only applied for Path ID 0.
 
 A MAX_PATH_ID frame (type=0x15228c0c) informs the peer of the maximum path identifier
 it is permitted to use.
+If the number of allocated paths has reached the local limit of max path identifier supported
+by the endpoint, it SHOULD send MAX_PATH_ID frame to increase the local limit and inform the peer.
```

Comment:
I don`t think we should make a normative requirement here. it's really the receiver decision what to do here.

Reply:
I agree with @mirjak. We cannot have an unqualified SHOULD here. If the peer has reached its internal active path limit, it is not going to increase MAX_PATH_ID. The client endpoint has a budget. If the peer only supports N path, it is not going to increase that to N+1. If the client wants to open a new path, it has to abandon an old one.

Reply:
Okay. I removed normative language here.